src/lucky_record/form.cr

```diff
@@ -165,6 +165,27 @@ abstract class LuckyRecord::Form(T)
 
   macro allow(*field_names)
     {% for field_name in field_names %}
+      {% if field_name.is_a?(TypeDeclaration) %}
```

Comment:
If you do something like `allow name : String` it will tell you to do `allow name` instead :)